client/src/components/Input.js

```diff
@@ -0,0 +1,17 @@
+import styled from '@emotion/styled';
+
+const Input = styled.input`
+  background-color: #efeaea;
+  color: #4d4d4d;
+  border-radius: 5rem;
+  border: none;
+  text-align: left;
+  box-shadow: 0rem 0.2rem 0.4rem rgba(172, 109, 26, 56);
+  font: Roboto Mono 1rem regular;
```

Comment:
it would be easier to have the font set in the `GlobalStyles`, especially the `font-size`, since you are using `rem` for a lot of other values as well

Reply:
Thank you. I changed it.